common/service/service.go

```diff
@@ -46,39 +44,30 @@ type (
 	// BootstrapParams holds the set of parameters
 	// needed to bootstrap a service
 	BootstrapParams struct {
-		Name            string
-		Logger          bark.Logger
-		MetricScope     tally.Scope
-		RingpopFactory  RingpopFactory
-		TChannelFactory TChannelFactory
-		CassandraConfig config.Cassandra
-	}
-
-	// TChannelFactory creates a TChannel and Thrift server
-	TChannelFactory interface {
```

Comment:
fyi - this change will also break the internal cadence-server wrapper around github.com/uber/cadence

Reply:
i have a pending CR for that.